ykrt/build.rs

```diff
@@ -1,13 +1,20 @@
+use std::env;
+
 pub fn main() {
     ykbuild::apply_llvm_ld_library_path();
-
+    println!("cargo:rerun-if-env-changed=YKB_TRACER");
     // Always compile in the LLVM JIT compiler.
     println!("cargo:rustc-cfg=jitc_llvm");
     // Always compile in our bespoke JIT compiler.
     println!("cargo:rustc-cfg=jitc_yk");
-    // Always compile in the HWT tracer.
-    println!("cargo:rustc-cfg=tracer_hwt");
     // FIXME: This is a temporary hack because LLVM has problems if the main thread exits before
     // compilation threads have finished.
     println!("cargo:rustc-cfg=yk_llvm_sync_hack");
+    match env::var("YKB_TRACER") {
+        Ok(ref tracer) if tracer == "swt" => println!("cargo:rustc-cfg=tracer_swt"),
```

Comment:
Can this be `Ok("swt")` instead or won't that work?

Reply:
I think the only way to do so will be to declare "swt" and "hwt" as variables and then match on that.

If I use `Ok("swt")` I get type mismatch: `expected String, found &str`
And I can't use `String::from` cause calls are not allowed in patterns.

Reply:
You might be able to do `match env::var("YKB_TRACER").as_ref()` but it's not that important I guess. I leave it up to you if you want to try.

Reply:
That gives me an error as well: `expected &String, found &str`
I guess I could do something like:
```
match env::var("YKB_TRACER").as_ref().map(|s| s.as_str()) {
    Ok("swt") => println!("cargo:rustc-cfg=tracer_swt"),
     ...
}
```
But its also not great :)

Reply:
Okay, nevermind. Let's not get held up by this. :-)